File: Fase 1/src/guia01/Ejercicio01.cpp
#include <iostream>
using namespace std;

const int N = 5;

int sumarArreglo(int arreglo[], int n) {
    int suma = 0;
    for (int i = 0; i < n; i++) {
        suma += arreglo[i];
    }
    return suma;
}

int main() {
    int arreglo[N] = {1, 2, 3, 4, 5};
    int suma = sumarArreglo(arreglo, N);
    cout << "La suma de los elementos del arreglo es: " << suma << endl;
    return 0;
}

File: Fase 1/src/guia01/Ejercicio01.java
package guia01;
/*
1. Crea una función en Python, C++ y Java que reciba un arreglo como parámetro y devuelva la suma de todos los elementos del arreglo.
¿Cuáles son las diferencias entre los tres lenguajes respecto al envío de arreglos como parámetro?
*/
public class Ejercicio01 {
  public static void main(String[] args) {
    int[] arreglo = {1, 2, 3, 4, 5};
    int suma = sumarArreglo(arreglo);
    System.out.println("La suma de los elementos del arreglo es: " + suma);
  }

  public static int sumarArreglo(int[] arreglo) {
    int sum = 0;
    for (int i = 0; i < arreglo.length; i++) {
      sum += arreglo[i];
    }
    return sum;
  }
}
